src/components/structures/RoomStatusBar.js

```diff
@@ -53,6 +51,10 @@ module.exports = React.createClass({
         // more interesting)
         hasActiveCall: React.PropTypes.bool,
 
+        // Number of names to display in typing indication. E.g. set to 3, will
+        // result in "X, Y, Z and 100 others are typing."
+        whoIsTypingLimit: React.PropTypes.number,
```

Comment:
Should probably put a default in for this, especially if it's not marked as required.